### ISSUE TITLE
Adding new members to a project - specifying Institution

### ISSUE THREAD

**Issue by jjkoehorst** [seek4science/seek#547]
**Describe the bug**
When adding a new member to a project the first step is to select the person and next is the institute.
I just tried it by adding Alan and the Indian institute and without his input he is suddenly not only part of manchester but also part of the Indian institute which obviously is wrong.

**To Reproduce**
1. Go to 'Project'
2. Click on 'Actions, Administer project members'
3. Add a person select a random institute
4. submit

**Expected behavior**
Prevention from adding a nonsense institute to a person

**Additional context**
I am curious what the reason is behind step 2 as why does the project owner need to know from which institute a person is from to add it to a project? Should this information not be only available at the person's profile page?

**Comment by alaninmcr** [comment]
I agree. A person should be able to specify their institutions and then the drop down can only select from their institutions

**Comment by stuzart** [comment]
we have cases where a person can belong to different institutions in the context of different projects, and it is not that uncommon or nonsensible. For example we have somebody that is in Manchester for one project, Netherlands for another, and South Africa for another. There are other cases of people in different countries, or different institutes within the same country. It is assumed that the project administrator knows the person that they are adding.
In the new 1.11 version, a Person can specify the institution themselves, either existing or new, when requesting to join
